Add `osu!catch world cup 2026: Registrations Now Open!` news post and accompanying wiki article (#14673)
ppy/osu-wiki: news/2026/2026-04-30-osucatch-world-cup-2026-registrations-now-open.md

---
layout: post
title: "osu!catch World Cup 2026: Registrations Now Open!"
date: 2026-04-30 18:00:00 +0000
series: world_cups
---

Our premier fruit-catching competition returns – The osu!catch World Cup is back with a slew of changes.

![](/wiki/Tournaments/CWC/2026/img/cwc2026-banner.jpg)

> [![Click here to register for CWC 2026](/wiki/shared/news/2026-04-30-osucatch-world-cup-2026-registrations-now-open/register-news.png)](https://osu.ppy.sh/community/tournaments/55)

Reviewer: move this below `## How do I register?`

## Format changes

> The 2026 osu!catch World Cup will introduce the protect system to the regular pick/ban phase. In this system, each team has to protect one beatmap before the ban phase can take place. This ensures that each team can play one map of their choice, which creates more avenues for strategizing.

Reviewer: :gb:

```suggestion
The 2026 osu!catch World Cup will introduce the protect system to the regular pick/ban phase. In this system, each team has to protect one beatmap before the ban phase can take place. This ensures that each team can play one map of their choice, which creates more avenues for strategising.
```

## Mappool changes

> This year's mappool structure introduces several adjustments meant to refine core catch skillsets and improve map quality and consistency. As such, we've opted to remove the gimmick (often easily circumvented by lane covers) and convert slots (difficult to source consistently), as well as MM3 (which was removed in favour adding NM5), respectively.

Reviewer: "respective" doesn't make sense here

```suggestion
This year's mappool structure introduces several adjustments meant to refine core osu!catch skill sets and improve map quality and consistency. As such, we've opted to remove the gimmick (often easily circumvented by lane covers) and convert slots (difficult to source consistently), as well as MM3 (which was removed in favour of adding NM5).
```

> To replace these, the No Mod pool now focuses on more fundamental mechanics, with NM1 focusing on streams, NM3 on jumps, and the newly added NM5 being a speed pick. NM4 was changed into a slot for "convert-like" patterns  . For the Double Time bracket, DT1 is now a dedicated stamina pick, moving high AR to DT3 to better pair with the streams slot, which naturally has a higher density (in most cases).

Reviewer 2: ```Suggestion
To replace these, the No Mod pool now focuses on more fundamental mechanics, with NM1 focusing on streams, NM3 on jumps, and the newly added NM5 being a speed pick. NM4 was changed into a slot for "convert-like" patterns. For the Double Time bracket, DT1 is now a dedicated stamina pick, moving high AR to DT3 to better pair with the streams slot, which naturally has a higher density (in most cases).
```

Please refer to the tables below for a more in-depth breakdown of the mappool changes:

> **Qualifier stage**

Reviewer: should look more proper

```suggestion
### Qualifier stage
```

![CWC qualifier pool comparison](/wiki/shared/news/2026-04-30-osucatch-world-cup-2026-registrations-now-open/quals.png)

> **Bracket stage**

Reviewer: ```suggestion
### Bracket stage
```

![CWC bracket pool comparison](/wiki/shared/news/2026-04-30-osucatch-world-cup-2026-registrations-now-open/bracket.png)

---

For more details pertaining to the tournament, refer to the **[osu!catch World Cup 2026 wiki article](/wiki/Tournaments/CWC/2026)**. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/2201978)** or the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.

## Tournament schedule

| Event | Date |
| --: | :-- |
| Registration phase | Apr 30 (18:00 UTC) – May 15 (23:59 UTC) |
| Qualifier showcase | May 24 (14:00 UTC) |
| Qualifier stage | May 30 & May 31 |
| Round of 32 | Jun 6 & Jun 7 |
| Round of 16 | Jun 13 & Jun 14 |
| Quarterfinals | Jun 20 & Jun 21 |
| Semifinals | Jun 27 & Jun 28 |
| Finals | Jul 4 & Jul 5 |
| Grand Finals | Jul 11 & Jul 12 |

## Prizes

The osu!catch World Cup 2026 offers a $2,000 prize pool, along with limited-edition merch!

> The 1st place team will receive 50% of the prize pool, limited-edition merch, a profile badge, and the **osu!catch Champion** user title for one year. The 2nd and 3rd place teams will receive 30% and 20% of the prize pool respectively, alongside limited-edition merch and a profile badge.

Reviewer: ```suggestion
The 1st-place team will receive 50% of the prize pool, limited-edition merch, a profile badge and the **osu!catch Champion** user title for one year. The 2nd- and 3rd-place teams will receive 30% and 20% of the prize pool respectively, alongside limited-edition merch and a profile badge.
```
